File: NewAdsExamples/app/src/main/java/com/cbitss/newadsexamples/MainActivity.java
package com.cbitss.newadsexamples;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import com.google.android.gms.ads.AdListener;
import com.google.android.gms.ads.AdRequest;
import com.google.android.gms.ads.AdView;
import com.google.android.gms.ads.LoadAdError;
import com.google.android.gms.ads.MobileAds;
import com.google.android.gms.ads.initialization.InitializationStatus;
import com.google.android.gms.ads.initialization.OnInitializationCompleteListener;

public class MainActivity extends AppCompatActivity {
private AdView mAdView;
Button movetofllscreen,movetovideoadd;
    private static final String TAG = "MainActivity";
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        mAdView = findViewById(R.id.adView);
        movetofllscreen = findViewById(R.id.movetoinsertiealadd);
        movetovideoadd = findViewById(R.id.movetovideoadd);
        movetovideoadd.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this,VideoAdd.class));
            }
        });
        movetofllscreen.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                startActivity(new Intent(MainActivity.this,FullScreenAdd.class));
            }
        });
        MobileAds.initialize(getApplicationContext(), new OnInitializationCompleteListener() {
            @Override
            public void onInitializationComplete(@NonNull InitializationStatus initializationStatus) {
                Toast.makeText(getApplicationContext(), "mobile is ready for ads testing", Toast.LENGTH_SHORT).show();
              AdRequest adRequest = new AdRequest.Builder().build();
                mAdView.loadAd(adRequest);

            }
        });
        mAdView.setAdListener(new AdListener() {
            @Override
            public void onAdClosed() {
                super.onAdClosed();
                Log.d(TAG, "onAdClosed: ");
            }

            @Override
            public void onAdFailedToLoad(@NonNull LoadAdError loadAdError) {
                super.onAdFailedToLoad(loadAdError);
                Log.d(TAG, "onAdFailedToLoad: ");

            }

            @Override
            public void onAdOpened() {
                super.onAdOpened();
                Log.d(TAG, "onAdOpened: ");
            }

            @Override
            public void onAdLoaded() {
                super.onAdLoaded();
                Log.d(TAG, "onAdLoaded: ");
            }

            @Override
            public void onAdClicked() {
                super.onAdClicked();
                Log.d(TAG, "onAdClicked: ");
            }

            @Override
            public void onAdImpression() {
                super.onAdImpression();
                Log.d(TAG, "onAdImpression: ");
            }
        });
    }
}

File: ReccylerViewComponent/app/src/main/java/com/cbitss/reccylerviewcomponent/MainActivity.kt
package com.cbitss.reccylerviewcomponent

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView

class MainActivity : AppCompatActivity() {

   lateinit var itemlist:MutableList<ItemModel>
   lateinit var recyclercompointer:RecyclerView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        recyclercompointer = findViewById(R.id.recyclervew)

        var layoutManager =LinearLayoutManager(applicationContext)
        recyclercompointer.layoutManager = layoutManager
        itemlist = ArrayList<ItemModel>()
        intiselist()


var adapteris = RecyclerAdapter(itemlist,applicationContext)
recyclercompointer.adapter = adapteris

    }

    private fun intiselist() {
        itemlist.add(ItemModel(R.drawable.bookone,"5435345"))
        itemlist.add(ItemModel(R.drawable.booktwo,"s45345"))
        itemlist.add(ItemModel(R.drawable.bookthree,"087987"))
        itemlist.add(ItemModel(R.drawable.sky,"5435345"))
        itemlist.add(ItemModel(R.drawable.univers,"5435345"))
        itemlist.add(ItemModel(R.drawable.bookone,"5435345"))
        itemlist.add(ItemModel(R.drawable.booktwo,"s45345"))
        itemlist.add(ItemModel(R.drawable.bookthree,"087987"))
        itemlist.add(ItemModel(R.drawable.sky,"5435345"))
        itemlist.add(ItemModel(R.drawable.univers,"5435345"))
        itemlist.add(ItemModel(R.drawable.bookone,"5435345"))
        itemlist.add(ItemModel(R.drawable.booktwo,"s45345"))
        itemlist.add(ItemModel(R.drawable.bookthree,"087987"))
        itemlist.add(ItemModel(R.drawable.sky,"5435345"))
        itemlist.add(ItemModel(R.drawable.univers,"5435345"))
        itemlist.add(ItemModel(R.drawable.bookone,"5435345"))
        itemlist.add(ItemModel(R.drawable.booktwo,"s45345"))
        itemlist.add(ItemModel(R.drawable.bookthree,"087987"))
        itemlist.add(ItemModel(R.drawable.sky,"5435345"))
        itemlist.add(ItemModel(R.drawable.univers,"5435345"))
    }
}